packages/rx_bloc_cli/lib/src/processors/ios/podfile_processor.dart

```diff
@@ -70,10 +70,15 @@ class PodfileProcessor extends StringProcessor {
   }
 
   void _addBuildConfigurations(StringBuffer buffer) {
-    String deploymentTargetsConfig;
-    deploymentTargetsConfig = '''\n
+    final deploymentTargetsConfig = '''\n
     target.build_configurations.each do |config|
         config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '$_minSupportedIOSVersion'
+        ${args.qrScannerEnabled ? '''
```

Comment:
Please make sure the fix is mention in the change log

Reply:
Done